oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtIssuerAuthenticationManagerResolver.java

```diff
@@ -130,9 +130,28 @@ public AuthenticationManager resolve(HttpServletRequest request) {
 		return authenticationManager;
 	}
 
+	/**
+	 * Set a custom issuer converter
```

Comment:
Will you please update the JavaDoc so that it talks about `BearerTokenResolver` instead?

Reply:
Done.